src/main/java/raisetech/studet/management/controller/StudentController.java

```diff
@@ -33,11 +35,12 @@ public StudentController(StudentService service, StudentConverter converter) {
 
     //全件取得
     @GetMapping("/studentList")
-    public List<StudentDetail> getStudentList(){
+    public String getStudentList(Model model){
         List<Student> students = service.searchStudentList();
         List<StudentCourse> studentCourses = service.searchStudentCourse();
 
-        return converter.convertStudentDetails(students, studentCourses);
+        model.addAttribute("studentList",converter.convertStudentDetails(students,studentCourses));
```

Comment:
フォーマットしましょう
手動ではなくフォーマッターで行いましょう
```suggestion
        model.addAttribute("studentList", converter.convertStudentDetails(students,studentCourses));
```